tests/testthat/test-table_with_settings_ui.R

```diff
@@ -1,23 +1,118 @@
-testthat::test_that("Table with settings: UI screenshots", {
-  app <- shinytest2::AppDriver$new(
-    app_tws(),
-    name = "tws",
-    variant = "app_tws_ui",
-  )
-  app$set_inputs(`table_with_settings-downbutton-file_name` = "table")
-
-  # click on download button
-  app$click("table_with_settings-downbutton-dwnl")
-
-  # test clicking on modal
-  app$click("table_with_settings-expand")
-  # wait for the expand to happen
-  Sys.sleep(0.1)
-  app$set_inputs(`table_with_settings-modal_downbutton-lpp` = 70)
-  app$click("table_with_settings-modal_downbutton-dwnl")
-  app$set_inputs(`table_with_settings-modal_downbutton-file_name` = "table")
-
-  # now test values in json
-  app$expect_values(screenshot_args = FALSE, name = "final_values")
-  app$stop()
+testthat::test_that("table_with_settings_ui returns `shiny.tag.list`", {
+  testthat::expect_s3_class(table_with_settings_ui("STH"), "shiny.tag.list")
 })
+
+testthat::test_that(
+  "e2e: teal.widgets::table_with_settings is initialized with two buttons and a table",
```

Comment:
Please analyse what I suggested in plot_with_settings test and please apply similar to table_with_settings. This test is too long.